packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenService.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <UMCore/UMSingletonModule.h>
+#import <EXSplashScreen/EXSplashScreenController.h>
+#import <EXSplashScreen/EXSplashScreenViewProvider.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Entry point for handling SplashScreen associated mechanism.
+ * This class has state based on the following relation { ViewController -> ApplicationSplashScreenState }.
+ * Each method call has to be made using ViewController that holds Application's view hierachy.
+ */
+@interface EXSplashScreenService : UMSingletonModule <UIApplicationDelegate>
+
+/**
+ * Overloaded method. See main method below.
+ */
+- (void)show:(UIViewController *)viewController resizeMode:(EXSplashScreenImageResizeMode)resizeMode;
```

Comment:
`[self show:rootViewController resizeMode:…]` doesn't communicate what the method actually does (one could say it shows the view controller!). I would consider renaming it to something more describing